Source/IntegrationTests/TestFiles/LitTests/LitTest/logger/ProofDependencies.dfy

```diff
@@ -0,0 +1,453 @@
+// RUN: %diff "%s" "%s"
```

Comment:
nit: is there a reason there's so many empty lines at the start of this file?

Reply:
It's because those lines used to be comments and the expected messages in some tests have line numbers based on the contents of this file before removing those comments. I should really update the test to remove the blank lines, and re-number the lines in the expected test output, but that'll be a bit time-consuming, so I haven't yet.